pyproject.toml

```diff
@@ -58,6 +58,7 @@ motile-tracker = "motile_tracker:napari.yaml"
 "Documentation" ="https://funkelab.github.io/motile_tracker/"
 
 [tool.setuptools_scm]
+write_to = "src/motile_tracker/_version.py"
```

Comment:
@cmalinmayor supposedly if the _version.py is generated no git is required on the machine where this is installed

Reply:
That sounds right - I think the problem came when they downloaded a zip from git manually instead of a wheel. This should be taken care of by the bundling; I'll ask my collaborator to test on her machines when we are ready.